docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,206 @@
+.. warning::
+    This is early access alpha functionality. While usable, this has not yet been tested in a large number of different
+    scenarios. The API is still be subject to change *without any prior deprecation notice*! If you use this
+    functionality keep a close eye on the changelog for any changes that may affect your analysis.
+
+Kymotracking
+============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+
+Kymographs are a vital tool in understanding binding events on DNA. They are generated using one-dimensional scans
+over time. Moving particles can then be seen moving along the spatial coordinate in time. It is often desired to
+to track the position of such particles in time. Pylake comes with a few features that enable you to do so.
+
+
+Performing the tracking
+-----------------------
+
+Performing Kymotracking is usually performed in two steps. Typically, there is an image processing step, following by
+a tracing step, where the actual traces are found.
+
+
+Using the greedy algorithm
+--------------------------
+
+First, we need to get an image to perform the tracing on. Let's grab a kymograph from a Bluelake file::
+
+    file = lk.File('kymograph.h5')
+    _, kymo = file.kymos.popitem()
```

Comment:
Keep the name here even if we're not using it. This could be someone's first time seeing this pattern.
```suggestion
    name, kymo = file.kymos.popitem()
```

Reply:
Good point.